File: arrays/500. Keyboard Row/findWords.js
/**
 * @param {string[]} words
 * @return {string[]}
 */
var findWords = function(words) {
    let result = [];
    const firstRow = "qwertyuiop";
    const secondRow = "asdfghjkl";
    const thirdRow = "zxcvbnm";

    for (let word of words) {
        // Turn it into a list so every can be used
        let lowercaseWord = word.toLowerCase().split("")
        if (lowercaseWord.every(char => firstRow.includes(char)) ||
                lowercaseWord.every(char => secondRow.includes(char)) ||
                lowercaseWord.every(char => thirdRow.includes(char)) ) {
            result.push(word);
        }
    }
    
    return result;
};

File: arrays/500. Keyboard Row/findWords.py
class Solution(object):
    def findWords(self, words):
        """
        :type words: List[str]
        :rtype: List[str]
        """
        result = []
        first_row = "qwertyuiop"
        second_row = "asdfghjkl"
        third_row = "zxcvbnm"

        for word in words:
            lowercase_word = word.lower()
            if (all(char in first_row for char in lowercase_word) or
                    all(char in second_row for char in lowercase_word) or
                    all(char in third_row for char in lowercase_word)):
                result.append(word)
        
        return result